HCIDOCS-651: Oracle Cloud C3 (#88897)
openshift/openshift-docs: installing/installing_oci/installing-c3-assisted-installer.adoc

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="installing-c3-assisted-installer"]
= Installing a cluster on {oci-c3-no-rt} by using the {ai-full}
:context: installing-c3-assisted-installer

toc::[]

{oci-c3} (C3) lets you run applications and middleware using {oci-first} services on high performance cloud infrastructure in your data center. 

// Overview
include::modules/c3-assisted-installer-overview.adoc[leveloffset=+1]

// Preparing the Bastian server
include::modules/c3-assisted-installer-preparing-bastion-server.adoc[leveloffset=+1]

// Terraform script execution - part 1
include::modules/c3-assisted-installer-running-script-via-home.adoc[leveloffset=+1]

// Preparing the OpenShift image
[id="c3-assisted-installer-preparing-image_{context}"]
== Preparing the {oci} image 

> Generate the {product-title} ISO image in the {ai-full} on the Red Hat portal. Then convert it to an {oci-first-no-rt} compatible image and upload it to the *Custom Images* page of your {oci-c3-no-rt} environment. 

Reviewer: ```suggestion
Generate the {product-title} ISO image in the {ai-full} on the Red{nbsp}Hat portal. Then, convert the image to an {oci-first-no-rt} compatible image and upload it to the *Custom Images* page of your {oci-c3-no-rt} environment. 
```
SSG: https://redhat-documentation.github.io/supplementary-style-guide/#non-breaking-spaces

> You can perform this step on your laptop and not on the bastion server or within environments such as Oracle Solution Center. 

Reviewer: ISG: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#perform
```suggestion
You can generate, convert, and upload the image on your laptop and not on the bastion server or within environments such as Oracle Solution Center. 
```
Please check this revision for accuracy.

include::modules/c3-assisted-installer-preparing-image-generating.adoc[leveloffset=+2]

include::modules/c3-assisted-installer-preparing-image-converting.adoc[leveloffset=+2]

// Terraform script execution - part 2
include::modules/c3-assisted-installer-running-script-via-region.adoc[leveloffset=+1]

// Completing the installation

[id="c3-ai-completing-installation_{context}"]

> == Completing the installation using the {ai-full} web console

Reviewer: ```suggestion
== Completing the installation by using the {ai-full} web console
```

> When you have configured the infrastructure, the instances are now running and are ready to be registered with Red Hat. 

Reviewer: ```suggestion
After you configure the infrastructure, the instances are running and are ready to be registered with Red{nbsp}Hat. 
```

:_mod-docs-content-type: ASSEMBLY
include::_attributes/common-attributes.adoc[]
[id="installing-c3-assisted-installer"]
= Installing a cluster on {oci-c3-no-rt} by using the {ai-full}
:context: installing-c3-assisted-installer

toc::[]

{oci-c3} (C3) lets you run applications and middleware using {oci-first} services on high performance cloud infrastructure in your data center. You can quickly deploy applications using a rack-scale platform with optional GPU acceleration and infrastructure-as-code automation. 

// Overview
include::modules/c3-assisted-installer-overview.adoc[leveloffset=+1]

// Preinstallation considerations
include::modules/c3-assisted-installer-preinstallation-considerations.adoc[leveloffset=+1]

// Preparing the Bastian server
include::modules/c3-assisted-installer-preparing-bastion-server.adoc[leveloffset=+1]

// Terraform script execution - part 1

> include::modules/c3-assisted-installer-running-script-via-home.adoc[leveloffset=+1]

Reviewer 2: Not sure  , there are two documents 1 for asisted-service and another for agent base installer.
To be on the safe side . the links here should be for assisted-installer ?

https://www.oracle.com/a/otn/docs/compute_cloud_at_customer_assisted_installer.pdf?source=:em:nl:mt::::PCATP

Author: The link you added is for the CPA updates in Assisted Installer (next story).
The link to the C3 updates is: https://app.slack.com/client/E030G10V24F
